_data/other.yml

```diff
@@ -30,7 +30,7 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
-      doc: https://kb.meraki.com/knowledge_base/log-in-security-configuring-account-and-password-policies-for-dashboard-administrators
+      doc: https://documentation.meraki.com/MX-Z/Client_VPN/Using_Two-Factor_Auth_with_Client_VPN
```

Comment:
Might want to use this: https://documentation.meraki.com/zGeneral_Administration/Managing_Dashboard_Access/Using_Google_Authenticator_for_Two-factor_Authentication_in_Dashboard